Library/Homebrew/rubocops/all.rb

```diff
@@ -13,29 +13,30 @@
 end
 
 require_relative "io_read"
+require_relative "move_to_extend_os"
 require_relative "shell_commands"
-require_relative "platform"
 
-require_relative "formula_desc"
+# formula audit cops
+require_relative "bottle"
```

Comment:
I sorted the requires in each group, and added a comment clarifying with this group represents